hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/DeltaSync.java

```diff
@@ -867,18 +894,20 @@ private Pair<Option<String>, JavaRDD<WriteStatus>> writeToSink(JavaRDD<HoodieRec
    *
    * @return Instant time of the commit
    */
-  private String startCommit() {
+  private String startCommit(String instantTime, boolean retryEnabled) {
```

Comment:
Is this change related with the PR?

Reply:
yes, this is required since we need instant time upfront.